src/panels/config/repairs/ha-config-repairs.ts

```diff
@@ -168,9 +164,14 @@ class HaConfigRepairs extends LitElement {
       outline: none;
       text-decoration: underline;
     }
-    ha-list-item {
-      cursor: pointer;
-      font-size: 16px;
+    ha-list-item-new {
+      div[slot="start"] img {
```

Comment:
CSS nesting is too new to be used right now, browser support: https://caniuse.com/css-nesting

Reply:
@bramkragten We should be able to use post css for that support, right?

Reply:
Yeah, was talking with Wendelin about that already